packages/core/src/sequence.ts

```diff
@@ -35,11 +72,21 @@ export class Sequence {
       this.sendError(res, req, err);
     }
   }
-
+   /**
+   * Sends API result in HTTP response
```

Comment:
@ritch sendResponse() and sendError(..) are not exported methods .. hence they are not APIs. They are local wrapper functions which @bajtos implemented in this class. I will remove API docs for these methods since they are not APIs. If end-user implements their own sequence, they should be using writeResultToResponse(..) and logError(..) instead..